android-project/app/src/main/java/org/diasurgical/devilutionx/ImportActivity.java

```diff
@@ -60,7 +63,7 @@ private void importFile(Uri fileUri) {
 		if (fileUri == null)
 			return;
 
-		DocumentFile file = DocumentFile.fromSingleUri(getApplicationContext(), fileUri);
+		DocumentFile file = Objects.requireNonNull(DocumentFile.fromSingleUri(getApplicationContext(), fileUri));

```

Comment:
fromSingleUri only returns null if called on versions older then KitKat which we already guard against accessing this function, so simply throwing a null pointer exception at this point should never cause an issue.